components/layout/app-bar.jsx

```diff
@@ -25,8 +26,14 @@ const DashboardAppBar = ({ children, className, ...restProps }) => {
       </AppBar.Brand>
 
       {children}
+      <AppBar.Item>
+        {`${store.user.givenName} ${store.user.familyName} `}|{' '}
+        <Button className={styles.logoutButton} onClick={store.logout}>
```

Comment:
I would prefer this being pure link.

Reply:
How? It cannot be a pure link because the cookie won't be deleted.

Reply:
Link to `api.core.ac.uk/logout?continue=/logout.html`